bundles/AdminBundle/Resources/public/js/pimcore/document/document_language_overview.js

```diff
@@ -242,15 +242,16 @@ pimcore.document.document_language_overview = Class.create({
                         },
                         items: [{
                             xtype: "textfield",
+                            itemId: "title",
```

Comment:
I would also make it consistent for translation new document option here https://github.com/pimcore/pimcore/blob/10.x/bundles/AdminBundle/Resources/public/js/pimcore/document/document.js#L481

Reply:
@dvesh3 thanks, good input. I've added another commit for this.